packages/nextjs/app/builders/0x11Bc455A4f64F800f9Ceb37614f02b213f98591d/page.tsx

```diff
@@ -0,0 +1,81 @@
+import Image from "next/image";
+import { backgroundPattern } from "./backgroundPattern";
+import { NextPage } from "next";
+import { FaDiscord, FaGithub, FaLinkedin, FaTelegram } from "react-icons/fa";
+import { FaXTwitter } from "react-icons/fa6";
+
+const EmarcProfile: NextPage = () => {
+  const socialLinks = [
```

Comment:
You could have to `socialLinks` array outside of the component. It prevents unnecessary recreation on each render.